### ISSUE TITLE
Serial ports don't work on Turkish Windows

### ISSUE THREAD

**Issue by tresf** [qzind/tray#772]
Steps:

1. Download and install Windows 10 x64 20H2 Turkish edition
2. Install QZ Tray 2.1.2
3. Visit https://demo.qz.io
4. Click "List ports"

The following error is thrown:
```log
2021-03-17 15:41:45,343 [INFO] Allowed localhost to access serial ports
2021-03-17 15:41:45,360 [ERROR] Connection error
java.lang.RuntimeException: Cannot call method public void qz.ws.PrintSocketClient#onMessage(org.eclipse.jetty.websocket.api.Session, java.io.Reader) with args: [org.eclipse.jetty.websocket.common.WebSocketSession, org.eclipse.jetty.websocket.common.message.MessageReader]
	at org.eclipse.jetty.websocket.common.events.annotated.CallableMethod.unwrapRuntimeException(CallableMethod.java:93)
	at org.eclipse.jetty.websocket.common.events.annotated.CallableMethod.call(CallableMethod.java:75)
	at org.eclipse.jetty.websocket.common.events.annotated.OptionalSessionCallableMethod.call(OptionalSessionCallableMethod.java:68)
	at org.eclipse.jetty.websocket.common.events.JettyAnnotatedEventDriver$2.run(JettyAnnotatedEventDriver.java:227)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:782)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:914)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.UnsatisfiedLinkError: Could not load the jssc library: Couldn't load library library jssc
	at jssc.SerialNativeInterface.<clinit>(SerialNativeInterface.java:90)
	at jssc.SerialPortList.<clinit>(SerialPortList.java:43)
	at qz.utils.SerialUtilities.getSerialPorts(Unknown Source)
	at qz.utils.SerialUtilities.getSerialPortsJSON(Unknown Source)
	at qz.ws.PrintSocketClient.processMessage(Unknown Source)
	at qz.ws.PrintSocketClient.onMessage(Unknown Source)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.eclipse.jetty.websocket.common.events.annotated.CallableMethod.call(CallableMethod.java:70)
	... 5 more
```

**Comment by tresf** [comment]
Patch added to master which remedies this by forcing Locale.  d376835.  Will leave open until a permanent fix lands upstream.

**Comment by tresf** [comment]
Closed via ed78c4e, 8af0219.
